tests/kuttl/test-public-webservice-sessionaffinity/03-delete.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+delete:
+- apiVersion: v1
+  kind: Namespace
+  name: test-web-services
```

Comment:
Wrong namespace and clowdenv are being deleted

Reply:
AH